### PR TITLE
feat: add system unavailable fail reason

### PR DESCRIPTION
**Pull request by slee1996**
Jira Link: https://atomicfi.atlassian.net/browse/CONN-2972?atlOrigin=eyJpIjoiMjFkMTQwZjdhYmQzNGU5YmE2YjI4ODZlZDYyNDVhN2IiLCJwIjoiaiJ9

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -728,7 +728,8 @@ The authentication status of a [Task](#create-task) was updated. Possible `authe
 | `product-not-supported`      | The account did not support the product.                                                                                                                                         |
 | `distribution-not-supported` | The account did not support the distribution, e.g. they requested to add an account for a percentage of their paycheck, but can only do fixed amounts and remainder/net balance. |
 | `unknown-failure`            | We encountered an unexpected error.                                                                                                                                              |
-| `routing-number-not-supported` | The account did not support the routing number entered.
+| `routing-number-not-supported` | The account did not support the routing number entered.                                                                                                                        |
+| `system-unavailable`         | The system was unavailable.                                                                                                                                             |
```
Comment (ajmueller) [inline source/index.html.md:732]: @slee1996 don't forget the docs need to be built: https://atomicfi.slack.com/archives/CS4TKPANS/p1618434633034600

Comment (ajmueller) [reply source/index.html.md:732]: @slee1996 sorry, should have noticed this the first time around, but we often get questions from our customers what these generally mean. The description here pretty much just repeats the reason and doesn't add much value. What could we add here to add some more value? Take a look at the [current fail reasons docs](https://docs.atomicfi.com/#fail-reasons). The `distribution-not-supported` reason would be a good example.

Looking through our code, it seems there are a couple of common reasons why we may use this fail reason:
* Scheduled availability of the site (e.g. with the unemployment sites)
* Downtime, either scheduled or unexpected

Can you think of any others that would be worth mentioning?

Comment (slee1996) [reply source/index.html.md:732]: @ajmueller I think those cover the two biggest, one other that we might want to add is cases where the system is technically available, but the user cannot currently access it (e.g. on Oregon if you do a deposit switch you have to wait another 10 days before you can make further changes). But that's different enough that we might want to define a different fail reason for it.

Comment (ajmueller) [reply source/index.html.md:732]: @slee1996 agreed, that sounds like it should be a different fail reason.

Comment (slee1996) [reply source/index.html.md:732]: @ajmueller how do you like this phrasing? "The system was unavailable, e.g. the site is undergoing maintenance, or it is outside the window of scheduled availability for the site."

Comment (ajmueller) [reply source/index.html.md:732]: @slee1996 looks good, but the comma before "or" isn't necessary:

> "The system was unavailable, e.g. the site is undergoing maintenance or it is outside the window of scheduled availability for the site."

Comment (slee1996) [reply source/index.html.md]: @ajmueller thanks for catching that!

**Review by ajmueller — APPROVED** [review]

**Pull request merged**